pkg/cloudprovider/providers/gce/cloud/service.go

```diff
@@ -27,6 +27,13 @@ import (
 	ga "google.golang.org/api/compute/v1"
 )
 
+const (
+	// maxConsecutiveOperationGetErrors is the number of failed attempts an operation GET will be
+	// attempted. In terms of time, this is 50 * (pollingInterval + callDuration). Assuming
+	// pollingInterval=1.5s, callDuration=100ms: the max tolerated outage is 80 seconds.
+	maxConsecutiveOperationGetErrors = 50
```

Comment:
why not specify this in time.Duration and do that math in the code?

Reply:
To avoid unit testing with time or importing a clock implementation.